projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,45 @@
+:host {
+    display: inline-flex;
+    min-width: 2.25rem;
+    min-height: 2.25rem;
+    align-items: center;
+    justify-content: center;
+    color: var(--tui-primary);
```

Comment:
I noticed that figma has a slight gradient applied to the current color. We can add a slight tint with `::after` element with small gradient from transparent to slightly visible black.

Reply:
fixed